example/cumulus-tf/main.tf

```diff
@@ -93,7 +95,7 @@ module "cumulus" {
   async_operation_image = "${data.aws_ecr_repository.async_operation.repository_url}:${var.async_operation_image_version}"
 
   ecs_cluster_instance_image_id   = data.aws_ssm_parameter.ecs_image_id.value
-  ecs_cluster_instance_subnet_ids = length(var.ecs_cluster_instance_subnet_ids) == 0 ? var.lambda_subnet_ids : var.ecs_cluster_instance_subnet_ids
+  ecs_cluster_instance_subnet_ids = length(var.ecs_cluster_instance_subnet_ids) == 0 ? local.subnet_ids : var.ecs_cluster_instance_subnet_ids
```

Comment:
This can also be `lambda_subnet_ids` since you set that on line 89 https://github.com/nasa/cumulus/pull/2599/files#diff-192847ae3f8b97c1193324513f1a0e04d4f474bcf998b911a8a34370931c01c7R89